src/SLCore.UnitTests/SLCore.UnitTests.csproj

```diff
@@ -13,4 +13,8 @@
     <ProjectReference Include="..\TestInfrastructure\TestInfrastructure.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="FluentAssertions" Version="6.12.0" />
```

Comment:
I updated it for only one project, but maybe I should have done it for https://github.com/SonarSource/sonarlint-visualstudio/blob/86d1c450a50aa31bb3390f9a124390f01fa73337/src/SonarLint.Test.props

Reply:
Yes, you should do it there instead. We don't want to maintain in many places. Also why did you need to change version?

Reply:
I needed it for a feature that was not available in the old one